ui/warlock/inputs.ts

```diff
@@ -55,6 +55,12 @@ export const PetInput = InputHelpers.makeSpecOptionsEnumIconInput<Spec.SpecWarlo
 	changeEmitter: (player: Player<Spec.SpecWarlock>) => player.changeEmitter,
 });
 
+export const NewDPBehaviour = InputHelpers.makeSpecOptionsBooleanInput<Spec.SpecWarlock>({
+	fieldName: 'newDPBehaviour',
+	label: 'New DP behaviour',
+	labelTooltip: 'Changes DP to always be 10% of spell power, 1s ICD',
```

Comment:
```suggestion
	labelTooltip: 'Updates Demonic Pact calculation to new behavior introduced in 3.4.3: new potential pact is always 10% of spell power and applied on 1 second ICD (rather than 5 seconds)',
```

nit: can we elaborate a little more?